src/caret_analyze/plot/bokeh/histogram.py

```diff
@@ -0,0 +1,51 @@
+from bokeh.plotting import figure, show
+
+from caret_analyze.exceptions import UnsupportedTypeError
```

Comment:
Copyright is missing.

Reply:
@hsgwa I have added the copyright to the file.